### ISSUE TITLE
Fix toggle to work with colour contrasts

### ISSUE THREAD

**Issue by immattdavison** [immattdavison/NoMoreDomains#51]
The on/off toggle currently has bad colour contrast to the background - the toggle should be changed to work better with the background colour.

**Comment by Priyansh61** [comment]
Hey @immattdavison  I would like to work on this issue ?

**Comment by immattdavison** [comment]
> Hey @immattdavison  I would like to work on this issue ?

I'll assign you now!

**Comment by Priyansh61** [comment]
Just a small doubt @immattdavison  Can we use some other background color instead of blue.

I guess the blue color can be replaced with a darker color so that it provides a good contrast to the toggle button, since the blue color is very eye attractive.

**Comment by immattdavison** [comment]
No I want to keep the blue, I think a better idea would be to change the toggle itself to work differently, take a look at the dark reader on/off toggle, I think that would be the ideal solution (darkreader.org)

**Comment by Priyansh61** [comment]
So on toggle can we change the whole color scheme like dark reader or we just need to change the toggle button color only?

**Comment by immattdavison** [comment]
No not change the colour theme but the layout, have an on side and off side and have the correct side filled 🤷‍♂️

**Comment by immattdavison** [comment]
Essentially @Priyansh61 I think the way to go would be to have the toggle switch like this 
![image](https://user-images.githubusercontent.com/80921257/193623221-48697c55-06d8-45cf-9ad1-9228cc38742d.png)
and then have the switch colour use the orangered colour and the background be `#007bff` as it currently is

**Comment by Priyansh61** [comment]
Got it ! Thanks @immattdavison  :smile:

**Comment by immattdavison** [comment]
> Hey! is this under HacktoberFest2022 ? If yes I would like to work on this.

Assigned
